src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -475,7 +475,7 @@ private static ThreeState TryGetBestResult(ArrayBuilder<MemberResolutionResult<T
                 {
                     int badArg = firstSupported.Result.BadArgumentsOpt[0];
                     IdentifierNameSyntax badName = arguments.Names[badArg];
-                    diagnostics.Add(ErrorCode.ERR_FunctionPointersCannotBeCalledWithNamedArguments, location);
+                    diagnostics.Add(ErrorCode.ERR_FunctionPointersCannotBeCalledWithNamedArguments, new SourceLocation(badName));
```

Comment:
`badName.Location`

Reply:
The pattern `new SourceLocation(badName)` is used multiple times in existing code in this file. Do you want to update all of them or just this new code?

I also don't see a Location property on IdentifierNameSyntax through the API doc:

https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntax.identifiernamesyntax?view=roslyn-dotnet#properties

So I'll use `GetLocation()` method.

```suggestion
                    diagnostics.Add(ErrorCode.ERR_FunctionPointersCannotBeCalledWithNamedArguments, badName.GetLocation());
```

Reply:
All `SyntaxNode`s have an internal `Location` API. Please use that: http://sourceroslyn.io/#Microsoft.CodeAnalysis/Syntax/SyntaxNode.cs,f38d78484e47462d.